### ISSUE TITLE
[Accessibility: ACC-270.1] Button: UI cannot be accessed

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12607]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Each & every UI, must be accessible using the keyboard.
Keyboard is the prerequisite for support of assistive technologies to enable users with disabilities to read and operate software applications, hence all the UIs must be accessible with keyboard.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Compact Tokenizer](https://sap.github.io/fundamental-ngx/#/core/token)
Within the table ‘Users’ against column header ‘Role’ the ‘X more’ Buttons are not accessible with Keyboard.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/82ddbb30-52fd-40b8-8e15-c89814b11ca8">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 this is an internal application and it should be handled by the application.
